src/types.ts

```diff
@@ -31,6 +31,34 @@ export enum Network {
   TESTNET = 'testnet'
 }
 
+export enum Chain {
+  ETHEREUM = 'ethereum',
+  MATIC = 'matic',
+  KLAYTN = 'klaytn',
+  BSC = 'bsc',
+  SOLANA = 'solana',
+  ARBITRUM = 'arbitrum',
+  ARBITRUM_NOVA = 'arbitrum_nova',
+  AVALANCHE = 'avalanche',
+  OPTIMISM = 'optimism',
+  BASE = 'base',
+  ZORA = 'zora',
+
+  // Testnets
+  SEPOLIA = 'sepolia',
+  RINKEBY = 'rinkeby',
+  MUMBAI = 'mumbai',
+  BAOBAB = 'baobab',
+  BSC_TESTNET = 'bsctestnet',
+  GOERLI = 'goerli',
+  SOLDEV = 'soldev',
+  ARBITRUM_GOERLI = 'arbitrum_goerli',
+  AVALANCHE_FUJI = 'avalanche_fuji',
+  OPTIMISM_GOERLI = 'optimism_goerli',
+  BASE_GOERLI = 'base_goerli',
+  ZORA_TESTNET = 'zora_testnet'
+}
```

Comment:
I am questioning if we want to maintain this list here. Seems ripe for drift from the actual values on OpenSea. Should we have it here or just have the type be "string"?

Reply:
String works. Updated. Looks like I only need to add to the Payload type.